### PR TITLE
[Maintenance] Remove enshrined/svg-sanitize package

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A
|-----------------|-----
| Branch?         | 1.14
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

**Review by Rafikooo — DISMISSED** [review]

File: src/Sylius/Bundle/CoreBundle/Resources/config/services/validators.xml [hunk 1]
```diff
@@ -133,6 +133,12 @@
             <argument>%sylius_core.max_int_value%</argument>
             <tag name="validator.constraint_validator" alias="sylius_max_integer" />
         </service>
+
         <service id="sylius.validator.max_integer" alias="Sylius\Bundle\CoreBundle\Validator\Constraints\MaxIntegerValidator" />
+
+        <service id="sylius.validator.allowed_mime_types" class="Sylius\Bundle\CoreBundle\Validator\Constraints\AllowedImageMimeTypesValidator">
```
Comment (Wojdylak) [inline src/Sylius/Bundle/CoreBundle/Resources/config/services/validators.xml]:
```suggestion
        <service id="sylius.validator.allowed_image_mime_types" class="Sylius\Bundle\CoreBundle\Validator\Constraints\AllowedImageMimeTypesValidator">
```

**Review by Wojdylak — DISMISSED** [review]

File: UPGRADE-1.14.md [hunk 2]
```diff
@@ -31,6 +31,22 @@
...
 
+2. Package `enshrined/svg-sanitize` has been removed from the project due to license incompatibility with Sylius. You can still use it in your project by adding it manually. Just add it to your `composer.json` file:
+
+    ```json
+    {
+        "require": {
+            "enshrined/svg-sanitize": "^0.16",
+        }
+    }
+    ```
```
Comment (GSadee) [inline UPGRADE-1.14.md]: IMO, change it to use `composer require enshrined/svg-sanitize`, it would be simpler

File: UPGRADE-1.14.md [hunk 3]
```diff
@@ -31,6 +31,22 @@
...
+   Also `svg` image support has been removed. If you want to use `svg` images in your project, add `image/svg+xml` to the list of allowed mime types in `CoreBundle` configuration.
```
Comment (GSadee) [inline UPGRADE-1.14.md]:
```suggestion
   Also, uploading `svg` image support has been removed. If you want to use `svg` images in your project, add `image/svg+xml` to the list of allowed mime types in the `CoreBundle` configuration.
```

File: src/Sylius/Behat/Context/Api/Admin/ManagingProductImagesContext.php [hunk 4]
```diff
@@ -220,6 +220,19 @@ public function iShouldBeNotifiedThatTheProductVariantDoesNotBelongToTheOwner(Pr
...
+    /**
+     * @Then I should be notified that svg file is not allowed
+     */
+    public function iShouldBeNotifiedThatSvgFileIsNotAllowed(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            sprintf(
+                'The file is not a valid image.',
+            ),
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Admin/ManagingProductImagesContext.php]:
```suggestion
            'The file is not a valid image.',
```

File: src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/LiipImageFiltersPass.php [hunk 5]
```diff
@@ -19,6 +19,12 @@
...
 use Symfony\Component\DependencyInjection\Reference;
 
+trigger_deprecation(
+    'sylius/sylius',
+    '1.14',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0.',
+    LiipImageFiltersPass::class,
+);
 /**
  * @internal
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/LiipImageFiltersPass.php:29]:
```suggestion
 * @internal
 *
 * @deprecated
```

File: src/Sylius/Bundle/CoreBundle/Twig/FilterExtension.php [hunk 6]
```diff
@@ -17,6 +17,13 @@
 use Liip\ImagineBundle\Templating\FilterExtension as BaseFilterExtension;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
+trigger_deprecation(
+    'sylius/sylius',
+    '1.14',
+    'The "%s" class is deprecated and will be removed in Sylius 2.0.',
+    FilterExtension::class,
+);
+
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Twig/FilterExtension.php:26]: Add a `@deprecated` in docblock

**Review by GSadee — DISMISSED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Michał! :tada:

**Pull request merged**